glide.yaml

```diff
@@ -2,23 +2,23 @@ package: github.com/openshift/apiserver-library-go
 import:
 # kube first
 - package: k8s.io/api
-  version: kubernetes-1.16.0
+  version: kubernetes-1.17.0-rc.2
 - package: k8s.io/apimachinery
-  version: kubernetes-1.16.0
+  version: kubernetes-1.17.0-rc.2
 - package: k8s.io/apiserver
-  version: kubernetes-1.16.0
+  version: kubernetes-1.17.0-rc.2
 - package: k8s.io/client-go
-  version: kubernetes-1.16.0
+  version: kubernetes-1.17.0-rc.2
 - package: k8s.io/component-base
-  version: kubernetes-1.16.0
+  version: kubernetes-1.17.0-rc.2
 - package: k8s.io/kubernetes
-  version: v1.16.0
+  version: v1.17.0-rc.2
 - package: k8s.io/klog
   version: v0.4.0
```

Comment:
I think this could be `v1.0.0`  to match https://github.com/kubernetes/kubernetes/blob/v1.17.0-rc.2/go.mod#L540